_includes/footer.html

```diff
@@ -1,18 +1,22 @@
 <!-- This will be our footer -->
-<footer class="footer mdl-mega-footer">
-  <div class="mdl-mega-footer__bottom-section">
-    <div class="mdl-mega-footer__drop-down-section">
-      <div class="mdl-mega-footer__heading" >Hack For LA</div>
-      <p class="footer_slogan">100 Automations is one of the many projects at Hack for LA, Code for America’s Los
-        Angeles chapter.
+<footer class="footer mdl-mini-footer">
+  <div class="footer-slogan">
+      <p class="">100 Automations is one of the many projects at Hack&nbsp;for&nbsp;LA, Code&nbsp;for&nbsp;America’s Los&nbsp;Angeles&nbsp;chapter.
```

Comment:
Very clever to use html entities to keep the names from breaking. I'm going to steal that.